packages/ga4/src/index.ts

```diff
@@ -9,8 +9,11 @@ import {
   isTargetElement,
   getUrlData,
   getEventParams,
+  mergeEventParams,
+  EventParamArray,
```

Comment:
**question**: This seems to be very similar to the following type, can we re-use that instead?

https://github.com/jahilldev/minimal-analytics/blob/1a323811d0fd590bd17a7dbf0a609909aed050d0/packages/shared/src/values.ts#L10